packages/fast-components-react-msft/src/carousel/carousel.spec.tsx

```diff
@@ -488,4 +486,44 @@ describe("carousel", (): void => {
             expect(mockLeave).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe("loop", (): void => {
+        const props: CarouselProps = {
+            ...handledProps,
+            managedClasses: {
+                carousel: "carousel",
+                carousel_flipperNext: "carousel_next",
+                carousel_flipperPrevious: "carousel_previous",
+            },
+        };
+
+        test("should initialize with `loop` defaulted to true", () => {
+            const rendered: any = mount(<MSFTCarousel {...props} />);
+
+            expect(rendered.props().loop).toBe(true);
+        });
+
+        test("should set `loop` to false when `loop` is passed as false", () => {
+            const rendered: any = mount(<MSFTCarousel {...props} loop={false} />);
+
+            expect(rendered.props().loop).toBe(false);
+        });
+
+        test("should not generate a previous flipper for first slide when `loop` is passed as false", () => {
+            const rendered: any = mount(<MSFTCarousel {...props} loop={false} />);
+
+            expect(rendered.state("activeId")).toBe("id01");
+            expect(rendered.exists(".carousel_next")).toBe(true);
+            expect(rendered.exists(".carousel_previous")).toBe(false);
+        });
```

Comment:
If the test doesn't already exist, you might want to also test that the flippers *do* exist when loop is true.

Reply:
loop defaults to true, and there is a flipper test that checks for default flippers showing in the, so that would you agree that those tests cover it?

Reply:
If the test isn't already written, I'd add it just to be safe IMO. If someone follows with a change we would want that case to be accounted for to ensure the test breaks

Reply:
tests added for clarity and to guarantee coverage